doc/source/building/build_in_buildservice.rst

```diff
@@ -195,3 +216,43 @@ setting is not sufficient. The following additional line is required:
 .. code:: bash
 
    Release: <CI_CNT>.<B_CNT>
+
+If you are building Vagrant images (see :ref:`setup_vagrant`) then also add
+the repository-type `vagrant`. OBS will then create a `boxes/` subdirectory
+in your download repositories, which will contain JSON files for Vagrant
+[#f4]_.
+
+
+If you have added your repositories to :file:`config.xml`, you will
+probably see a lot of errors of the following type:
+
+.. code:: bash
+
+   unresolvable: have choice for SOMEPACKAGE: SOMEPAKAGE_1 SOMEPACKAGE_2
+
+These can be worked around by adding a lot of ``Prefer:``, ``Substitute:``,
+``Preinstall:`` and ``Support:`` directives. If you do not want to put your
+repository configuration into OBS, then we recommend to initially copy the
+current project configuration of the testing project
+`Virtualization:Appliances:Images:Testing_$ARCH` into your own project as a
+start and tweak it from there instead of starting from scratch.
+
+
+.. [#f1] This is a design decision made by OBS: as it's purpose is to build
+   packages in a reproducible fashion it cannot make a decision which
+   package to choose from multiple available ones. A package manager build
+   for end-users on the other hand must make an a choice, as it would be
+   otherwise hardly usable.
+
+.. [#f2] :file:`osc` will compress added folders into `cpio
+   <https://en.wikipedia.org/wiki/Cpio>`_ archive and decompress it before
+   building running your build. The only downside of this is, that the
+   contents of your overlay are not conveniently visible via the web
```

Comment:
Either it's one overlay then "**is** not conveniently visible" or there are more overlay**s**. :wink:

Reply:
The "are" references the contents of the overlay, which there are usually more than one file. How would I write that correctly?

Reply:
Maybe like this:

```suggestion
 contents of your overlays are not conveniently visible via the web
```

Reply:
There's only one overlay, so that wouldn't be correct.

Reply:
Ok, I see. In that case, you could do:

```suggestion
contents of your overlay is not conveniently visible via the web
```